### PR TITLE
Fixed floating switch when set height and width for player

### PR DESCRIPTION
**Pull request by sambua**

### CODE REVIEW

File: src/dynamics/video_player/player/player.js [hunk 1]
```diff
@@ -775,15 +775,15 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
...
+                    "containerSizingStyles:aspect_ratio,height,width,is_floating,hideplayer,floatingoptions.floatingonly,fullscreened,gallerysidebar": function(
                         aspectRatio,
                         height,
                         width,
                         isFloating,
                         hidden,
                         floatingonly,
                         fullscreened,
-                        showSidebar
+                        gallerySidebar
```
Comment (ntshcalleia) [inline src/dynamics/video_player/player/player.js:786]: Hi @sambua, we're using showSidebar on line 792, so this breaks the player

File: src/dynamics/video_player/player/player.js [hunk 2]
```diff
@@ -827,6 +827,8 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
                                 // in other case width will be applied twice
                                 containerStyles.width = "100%";
                             }
+                            // will shrink the page
+                            if (gallerySidebar) delete containerStyles.aspectRatio;
```
Comment (ntshcalleia) [inline src/dynamics/video_player/player/player.js:831]: Hi @sambua, this breaks the activeElement's aspect ratio on transition from floating to inline. 😬

**Review by ntshcalleia — CHANGES_REQUESTED** [review]

File: src/dynamics/video_player/player/player.js (hunk 2 above)
Comment (sambua) [reply src/dynamics/video_player/player/player.js:831]: Hi @ntshcalleia how you can reproduce it? I have made some tests and this line will be only affect if it's with sidebar gallery, and no floating at all.

**Comment by ntshcalleia** [comment]
Hi @sambua, I attached a video recording. As you can see, the container size goes from 921x421.86 to 921x552.38 after switching to floating state:

https://github.com/betajs/betajs-media-components/assets/5845648/139f2cbb-b482-45a7-9152-b556c175171a

**Pull request merged**